q2cli/builtin/tools.py

```diff
@@ -7,14 +7,15 @@
 # ----------------------------------------------------------------------------
 
 import os
-
+import qiime2
```

Comment:
@thermokarst thanks for bringing this up! I'm wondering if the following modification will make a difference (in terms of import time):

`from qiime2.metadata.base import SUPPORTED_COLUMN_TYPES as _COLUMN_TYPES`

Does this prevent the entire qiime2 import from occurring?

Reply:
>  I'm wondering if the following modification will make a difference (in terms of import time)

This is a great suggestion, let's take some measurements!

<img width="1246" alt="Screen Shot 2021-07-20 at 1 25 03 PM" src="https://user-images.githubusercontent.com/274668/126390809-5a16e290-4d41-4304-8264-9afd86673148.png">

😞 

So to be fair, the framework isn't _that_ slow, the main source for slowness is when loading up an instance of the plugin manager (which you aren't doing here, fortunately). But, the precedent in this module is to defer import of `qiime2` until the last possible moment, because it could be slow.

Give the cache a shot and let me know how it goes - I think once you get your bearings it'll be relatively quick and easy.